src/components/Messages/Message.tsx

```diff
@@ -14,19 +15,17 @@ import { useAppStore } from 'src/store/app';
 import { useMessageStore } from 'src/store/message';
 
 const Message: FC = () => {
-  const { push, query } = useRouter();
-  const address = query.address as string;
+  const { query } = useRouter();
+  const address = (query.address as string).toLowerCase();
```

Comment:
We should be careful about casing here. The SDK is case-sensitive when looking for contacts and will show someone as not on the network if the casing is wrong (although we have an open issue to always convert to checksum case).

Reply:
Oooooh great callout! I want to guarantee a match between xmtp addresses and `profile.ownedBy` here since addresses are the only thing stitching the data together. Filed a follow-up to ensure/test case insensitivity: https://github.com/xmtp-labs/hq/issues/736.